.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
What is this file?

Reply:
This file is for VSCode and adds a `Jest tests` task that you can use via editor. That way the tests run in watch mode within the editor and you can add `debugger` statements